src/temp.bash

```diff
@@ -168,7 +168,7 @@ temp_del() {
 
   # Delete directory.
   local result
-  result="$(rm -r -- "$path" 2>&1)"
+  result="$(rm -rf -- "$path" 2>&1)"
```

Comment:
This instinctively makes me nervous : ] Should we add a check to ensure that `$path` doesn't equal `/` perhaps? Is there anything else we could do to protect from typos or accidental invocation?

Reply:
I tried what happens if you redirect `/dev/null` to `stdin` and it seems this is taken as implicit yes to the prompt. The prompt does not even show in this case and the return code is 0.